org/mozilla/jss/ssl/SSLCipher.java

```diff
@@ -365,10 +379,36 @@ public boolean isECC() {
         return ecc;
     }
 
+    public boolean isTLSv13() {
+        return tls13;
+    }
+
+    public boolean isSupported() {
+        return supported;
+    }
+
+    public boolean requiresRSACert() {
+        return this.name().contains("_RSA_");
+    }
+
+    public boolean requiresECDSACert() {
+        return this.name().contains("_ECDSA_");
+    }
+
     public static SSLCipher valueOf(int id) {
         for (SSLCipher cipher : SSLCipher.class.getEnumConstants()) {
             if (cipher.id == id) return cipher;
         }
         return null;
     }
+
+    public static SSLCipher fromName(String name) {
```

Comment:
Enums have built-in valueOf() so this method is redundant.
https://www.tutorialspoint.com/java/lang/enum_valueof.htm

Reply:
Ah sorry. I was thinking I was going to need to add aliases here (between IANA and JDK names -- kinda like `SSLVersion` has), but I don't think I need to actually. So I can remove this.